promscale/installation/source.md

```diff
@@ -0,0 +1,80 @@
+# Promscale from source
+You can install Promscale from source on any operating system. The Promscale
+source files are available from our
+[GitHub releases page][gh-promscale-download].
+
+Before you begin, you must have an already installed and working Prometheus
+environment. Additionally, you need a
+[self-hosted TimescaleDB instance][tsdb-install-self-hosted] installed.
+
+## Install Promscale from source
```

Comment:
I wouldn't call this "install from source", this is "Install precompiled binary"